### DISCUSSION TITLE
Drop Altay support?

### DISCUSSION THREAD

**Discussion by Endermanbugzjfc** [crnw-dev/IslandArchitect#27] [General]
# Should this plugin drop Altay support?
Originally, the organization of IslandArchitect (ClouriaNetwork) uses Altay as its server software. But since yesterday, the owner told me he will no longer uses Altay. So should I drop Altay support or not?

**Comment by ethaniccc** [comment]
If there's no other reason to support Atlay, I would say dropping support is the best option. 

(Below are for those that aren't on PMMP's Discord)
Arguments Thunder made:
```
what pros and cons do you get
more of "how painful is it to support it"
and "how much do you get for supporting it"
whichever wins
```

Arguments dylan made:
```
if you had to go out of your way to support it in the first place, i'd say it's already too much effort
```

**Comment by Mohagames205** [comment]
As Dylan already said, Altay should do their best to stay compatible with PM plugins and not the other way around. Supporting Altay will introduce more disadvantages than advantages.
